src/Notifications/AbstractNotification.php

```diff
@@ -27,17 +26,24 @@
     /**
      * TODO : is it an helper ?
      * Determine if a notification has been subscribed for a specific driver.
+     *
      * @param string $driver_id
+     *
      * @return bool
      */
-    final public static function isSubscribed(string $driver_id): bool
+    final public static function isSubscribed(string $driver_id = null) //: bool
```

Comment:
For which valid reason you may call this method without a `driver`?

Reply:
as explained on slack: `driver_id` for a user without driver specific `privateChannel` registered does not have a `driver_id`

Reply:
As the `driver` will determine the way
 - a notification is sent to a platform
 - an user is subscribing to receive private notification

I can't figure how it can be `null`?

Reply:
A user which does not have setup a driver does not have a `driver_id` 

f.e. this will return `null`
https://github.com/warlof/seat-notifications/pull/1/files#diff-fb5cf69377249583135e4fc608de534eR130

Reply:
ok i refactored it